### DISCUSSION TITLE
Atlas Brand related changes

### DISCUSSION THREAD

**Discussion by DevDroid1** [Atlas-OS/Atlas#1265] [Development]
this is not what you think it is
basically i have been using atlas os for a while now its good but its doesnt make my pc that much faster probably i have a general bad pc

i like all the features it adds/removes however one think i don't like is how it changes things like
- my username
- my pc name
- my wallpaper
- my profile picture
- my pc descirption (the thing that is under your pc name as soon as you open settings)

i would really like to know why this is and / or how i could change it either during install, with atlas configs or with a third party tool

**Comment by Ast3risk-ops** [comment]
> this is not what you think it is
> 
> basically i have been using atlas os for a while now its good but its doesnt make my pc that much faster probably i have a general bad pc
> 
> 
> 
> i like all the features it adds/removes however one think i don't like is how it changes things like
> 
> - my username
> 
> - my pc name
> 
> - my wallpaper
> 
> - my profile picture
> 
> - my pc descirption (the thing that is under your pc name as soon as you open settings)
> 
> 
> 
> i would really like to know why this is and / or how i could change it either during install, with atlas configs or with a third party tool

The team has responded to this a million times, the changes are *minor* (it does not change your PC name, description or username, you're just seeing the extra box atlas adds to show its version)

This is done so you know install was successful, and it's a good wallpaper 😁 (there's more in the artwork category)

Your wallpaper and profile picture can be changed manually with no effort, Atlas just replaces the very boring windows defaults for those.
